#!/bin/sh /usr/share/dpatch/dpatch-run
## 01-typos-and-spelling.dpatch by A Costa <agcosta@gis.net>
##
## DP: Fixes some typos and spelling mistakes.

@DPATCH@

================================================================================
--- cdparanoia-III-10.2/cdparanoia.1
+++ cdparanoia-III-10.2/cdparanoia.1
@@ -1,16 +1,17 @@
 .TH CDPARANOIA 1 "11 Sep 2008"
 .SH NAME
-cdparanoia 10.2 (Paranoia release III) \- an audio CD reading utility which includes extra data verification features
+cdparanoia \- an audio CD reading utility which includes extra data verification features
 .SH SYNOPSIS
 .B cdparanoia
 .RB [ options ]
 .B span 
 .RB [ outfile ]
+.RB | \-B
 .SH DESCRIPTION
 .B cdparanoia
-retrieves audio tracks from CDDA capable CDROM drives.  The data can
+retrieves audio tracks from CDDA-capable CDROM drives.  The data can
 be saved to a file or directed to standard output in WAV, AIFF, AIFF-C
-or raw format.  Most ATAPI, SCSI and several proprietary CDROM drive
+or raw format.  Most ATAPI and SCSI and several proprietary CDROM drive
 makes are supported;
 .B cdparanoia
 can determine if the target drive is CDDA capable.
@@ -69,7 +70,7 @@
 
 .TP
 .B \-p --output-raw
-Output headerless data as raw 16 bit PCM data with interleaved samples in host byte order.  To force little or big endian byte order, use 
+Output headerless data as raw 16-bit PCM data with interleaved samples in host byte order.  To force little or big endian byte order, use 
 .B \-r 
 or 
 .B \-R
@@ -77,40 +78,46 @@
 
 .TP
 .B \-r --output-raw-little-endian
-Output headerless data as raw 16 bit PCM data with interleaved samples in LSB first byte order.
+Output headerless data as raw 16-bit PCM data with interleaved samples in LSB first byte order.
 
 .TP
 .B \-R --output-raw-big-endian
-Output headerless data as raw 16 bit PCM data with interleaved samples in MSB first byte order.
+Output headerless data as raw 16-bit PCM data with interleaved samples in MSB first byte order.
 
 .TP
 .B \-w --output-wav
 Output data in Micro$oft RIFF WAV format (note that WAV data is always
-LSB first byte order).
+LSB-first byte order).
 
 .TP
 .B \-f --output-aiff
 Output data in Apple AIFF format (note that AIFC data is
-always in MSB first byte order).
+always in MSB-first byte order).
 
 .TP
 .B \-a --output-aifc
 Output data in uncompressed Apple AIFF-C format (note that AIFF-C data is
-always in MSB first byte order).
+always in MSB-first byte order).
 
 .TP
 .BI "\-B --batch "
 
-Cdda2wav-style batch output flag; cdparanoia will split the output
+Cdda2wav-style batch output flag;
+.B cdparanoia
+will split the output
 into multiple files at track boundaries.  Output file names are
 prepended with 'track#.'
 
 .TP
 .B \-c --force-cdrom-little-endian
 Some CDROM drives misreport their endianness (or do not report it at
-all); it's possible that cdparanoia will guess wrong.  Use
+all); it's possible that
+.B cdparanoia
+will guess wrong.  Use
 .B \-c
-to force cdparanoia to treat the drive as a little endian device.
+to force
+.B cdparanoia
+to treat the drive as a little endian device.
 
 .TP
 .B \-C --force-cdrom-big-endian
@@ -122,7 +129,7 @@
 .B n
 sectors per read.  This number can be misleading; the kernel will often
 split read requests into multiple atomic reads (the automated Paranoia
-code is aware of this) or allow reads only wihin a restricted size
+code is aware of this) or allow reads only within a restricted size
 range. 
 .B This option should generally not be used.
 
@@ -131,7 +138,7 @@
 Force the interface backend to read from
 .B device
 rather than the first readable CDROM drive it finds.  This can be used
-to specify devices of any valid interface type (ATAPI, SCSI or
+to specify devices of any valid interface type (ATAPI, SCSI, or
 proprietary).
 
 .TP
@@ -158,13 +165,13 @@
 may be used with
 .B \-d
 to explicitly set both the SCSI cdrom and
-generic (sg) devices seperately. This option is only useful on
+generic (sg) devices separately. This option is only useful on
 obsolete SCSI setups and when using the generic scsi (sg) driver. 
 
 .TP
 .BI "\-S --force-read-speed " number
 Use this option explicitly to set the read rate of the CD drive (where
-supported).  This can reduce underruns on machines with slow disks, or
+supported).  This can reduce underruns on machines which have slow disks, or
 which are low on memory.
 
 .TP
@@ -184,16 +191,19 @@
 and catching a bit of the next track).
 .B \-T
 accounts for this behavior.  Note that this option will cause
-cdparanoia to attempt to read sectors before or past the known user
+.B cdparanoia
+to attempt to read sectors before or past the known user
 data area of the disc, resulting in read errors at disc edges on most
 drives and possibly even hard lockups on some buggy hardware.
 
 .TP
 .BI "\-O --sample-offset " number
 Use this option to force the entire disc to shift sample position
-output by the given amount; This can be used to shift track boundaries
+output by the given amount; this can be used to shift track boundaries
 for the whole disc manually on sample granularity. Note that this will
-cause cdparanoia to attempt to read partial sectors before or past the
+cause
+.B cdparanoia
+to attempt to read partial sectors before or past the
 known user data area of the disc, probably causing read errors on most
 drives and possibly even hard lockups on some buggy hardware.
 
@@ -202,8 +212,11 @@
 .B \-Z --disable-paranoia
 Disable 
 .B all
-data verification and correction features.  When using -Z, cdparanoia
-reads data exactly as would cdda2wav with an overlap setting of zero.
+data verification and correction features.  When using -Z,
+.B cdparanoia
+reads data exactly as would
+.BR cdda2wav (1)
+with an overlap setting of zero.
 This option implies that 
 .B \-Y
 is active.
@@ -211,7 +224,9 @@
 .TP
 .B \-z --never-skip[=max_retries]
 Do not accept any skips; retry forever if needed.  An optional maximum
-number of retries can be specified; for comparison, default without -z is
+number of retries can be specified; for comparison, default without
+.B -z
+is
 currently 20.
 
 .TP
@@ -221,7 +236,7 @@
 
 .TP
 .B \-X --abort-on-skip
-If the read skips due to imperfect data, a scratch, whatever, abort reading this track.  If output is to a file, delete the partially completed file.
+If the read skips due to imperfect data, a scratch, or whatever, abort reading this track.  If output is to a file, delete the partially completed file.
 
 .SH OUTPUT SMILIES
 .TP
@@ -282,7 +297,9 @@
 .B
    !  
 Errors found after stage 1 correction; the drive is making the
-same error through multiple re-reads, and cdparanoia is having trouble
+same error through multiple re-reads, and
+.B cdparanoia
+is having trouble
 detecting them.
 .TP
 .B
@@ -295,8 +312,9 @@
 
 .SH SPAN ARGUMENT
 
-The span argument specifies which track, tracks or subsections of
-tracks to read.  This argument is required. 
+The span argument specifies which track, tracks, or subsections of
+tracks to read.  This argument is required,
+unless batch-mode is used (in bach-mode, cdparanoia will rip all traks if no span is given).
 .B NOTE:
 Unless the span is a simple number, it's generally a good idea to
 quote the span argument to protect it from the shell.
@@ -307,7 +325,7 @@
 1[ww:xx:yy.zz]-2[aa:bb:cc.dd] 
 .P
 Here, 1 and 2 are track numbers; the numbers in brackets provide a
-finer grained offset within a particular track. [aa:bb:cc.dd] is in
+finer-grained offset within a particular track. [aa:bb:cc.dd] is in
 hours/minutes/seconds/sectors format. Zero fields need not be
 specified: [::20], [:20], [20], [20.], etc, would be interpreted as
 twenty seconds, [10:] would be ten minutes, [.30] would be thirty
@@ -315,7 +333,7 @@
 .P
 When only a single offset is supplied, it is interpreted as a starting
 offset and ripping will continue to the end of the track.  If a single
-offset is preceeded or followed by a hyphen, the implicit missing
+offset is preceded or followed by a hyphen, the implicit missing
 offset is taken to be the start or end of the disc, respectively. Thus:
 
 .TP
@@ -335,7 +353,7 @@
 .B 2-4          
 Specifies ripping from the beginning of track 2 to the end of track 4.
 .P
-Again, don't forget to protect square brackets and preceeding hyphens from
+Again, don't forget to protect square brackets from
 the shell.
 
 .SH EXAMPLES
@@ -346,7 +364,7 @@
 .P
        cdparanoia -vsQ
 .TP
-Extract an entire disc, putting each track in a seperate file:
+Extract an entire disc, putting each track in a separate file:
 .P
        cdparanoia -B 
 .TP
@@ -354,26 +372,30 @@
 .P
        cdparanoia "1[:30.12]-1[1:10]"
 .TP
-Extract from the beginning of the disc up to track 3:
+Extract from the beginning of the disc up through track 3:
 .P
-       cdparanoia -- "-3"
+       cdparanoia -- -3
 .TP
 The "--" above is to distinguish "-3" from an option flag.
 .SH OUTPUT
 
 The output file argument is optional; if it is not specified,
-cdparanoia will output samples to one of
+.B cdparanoia
+will output samples to one of
 .BR cdda.wav ", " cdda.aifc ", or " cdda.raw
 depending on whether 
-.BR \-w ", " \-a ", " \-r " or " \-R " is used (" \-w 
+.BR \-w ", " \-a ", " \-r " or, " \-R " is used (" \-w 
 is the implicit default).  The output file argument of 
 .B \-
 specifies standard output; all data formats may be piped. 
 
 .SH ACKNOWLEDGEMENTS
-Cdparanoia sprang from and once drew heavily from the interface of
+.B cdparanoia
+sprang from and once drew heavily from the interface of
 Heiko Eissfeldt's (heiko@colossus.escape.de) 'cdda2wav'
-package. Cdparanoia would not have happened without it.
+package.
+.B cdparanoia
+would not have happened without it.
 .P
 Joerg Schilling has also contributed SCSI expertise through his
 generic SCSI transport library.
@@ -381,5 +403,6 @@
 .SH AUTHOR
 Monty <monty@xiph.org>
 .P
-Cdparanoia's homepage may be found at:
+.B cdparanoia's
+homepage may be found at:
 http://www.xiph.org/paranoia/
--- cdparanoia-III-10.2/main.c
+++ cdparanoia-III-10.2/main.c
@@ -51,7 +51,7 @@
 
   if(offset==NULL)return(-1);
 
-  /* seperate track from time offset */
+  /* separate track from time offset */
   temp=strchr(offset,']');
   if(temp){
     *temp='\0';
@@ -221,14 +221,14 @@
 "  -V --version                    : print version info and quit\n"
 "  -Q --query                      : autosense drive, query disc and quit\n"
 "  -B --batch                      : 'batch' mode (saves each track to a\n"
-"                                    seperate file.\n"
+"                                    separate file.\n"
 "  -s --search-for-drive           : do an exhaustive search for drive\n"
 "  -h --help                       : print help\n\n"
 
-"  -p --output-raw                 : output raw 16 bit PCM in host byte \n"
+"  -p --output-raw                 : output raw 16-bit PCM in host byte \n"
 "                                    order\n"
-"  -r --output-raw-little-endian   : output raw 16 bit little-endian PCM\n"
-"  -R --output-raw-big-endian      : output raw 16 bit big-endian PCM\n"
+"  -r --output-raw-little-endian   : output raw 16-bit little-endian PCM\n"
+"  -R --output-raw-big-endian      : output raw 16-bit big-endian PCM\n"
 "  -w --output-wav                 : output as WAV file (default)\n"
 "  -f --output-aiff                : output as AIFF file\n"
 "  -a --output-aifc                : output as AIFF-C file\n\n"
@@ -295,7 +295,7 @@
 "                       1[ww:xx:yy.zz]-2[aa:bb:cc.dd] \n\n"
 
 "Here, 1 and 2 are track numbers; the numbers in brackets provide a\n"
-"finer grained offset within a particular track. [aa:bb:cc.dd] is in\n"
+"finer-grained offset within a particular track. [aa:bb:cc.dd] is in\n"
 "hours/minutes/seconds/sectors format. Zero fields need not be\n"
 "specified: [::20], [:20], [20], [20.], etc, would be interpreted as\n"
 "twenty seconds, [10:] would be ten minutes, [.30] would be thirty\n"
@@ -303,7 +303,7 @@
 
 "When only a single offset is supplied, it is interpreted as a starting\n"
 "offset and ripping will continue to the end of he track.  If a single\n"
-"offset is preceeded or followed by a hyphen, the implicit missing\n"
+"offset is preceded or followed by a hyphen, the implicit missing\n"
 "offset is taken to be the start or end of the disc, respectively. Thus:\n\n"
 
 "    1:[20.35]    Specifies ripping from track 1, second 20, sector 35 to \n"
@@ -323,12 +323,15 @@
 "Don't forget to protect square brackets and preceeding hyphens from\n"
 "the shell...\n\n"
 "A few examples, protected from the shell:\n"
+"Don't forget to protect square brackets from the shell...\n\n"
+
+"A few examples, protected from the shell where appropriate:\n"
 "  A) query only with exhaustive search for a drive and full reporting\n"
 "     of autosense:\n"
 "       cdparanoia -vsQ\n\n"
-"  B) extract up to and including track 3, putting each track in a seperate\n"
+"  B) extract up to and including track 3, putting each track in a separate\n"
 "     file:\n"
-"       cdparanoia -B -- \"-3\"\n\n"
+"       cdparanoia -B -- -3\n\n"
 "  C) extract from track 1, time 0:30.12 to 1:10.00:\n"
 "       cdparanoia \"1[:30.12]-1[1:10]\"\n\n"
 
